tests/core/test_multi_derived_signal.py

```diff
@@ -131,3 +136,60 @@ def test_mismatching_args():
             jack22=soft_signal_rw(float),
             distance=soft_signal_rw(float),
         )
+
+
+@pytest.fixture
+def derived_signal_backend() -> SignalBackend[SignalDatatype]:
+    signal_r = soft_signal_rw(int, initial_value=4)
+
+    def _get(ts: int) -> float:
+        return ts
+
+    async def _put(value: float) -> None:
+        pass
+
+    derived = derived_signal_rw(_get, _put, ts=signal_r)
+    return derived._get_cache().backend
+
+
+async def test_derived_signal_backend_connect_pass(
+    derived_signal_backend: SignalBackend,
+):
+    result = await derived_signal_backend.connect(0.1)
+    assert result is None
```

Comment:
Then this would be
```suggestion
async def test_derived_signal_backend_connect_pass(
    derived_signal: SignalRW,
):
    result = await derived_signal.connect()
    assert result is None
```